zone/gm_commands/ucs.cpp

```diff
@@ -2,88 +2,87 @@
 
 void command_ucs(Client *c, const Seperator *sep)
 {
-	if (!c) {
+	LogInfo(
```

Comment:
Not sure how useful this is as this doesn't actually trigger a reconnect

Reply:
Yeah, me either, it literally does nothing from my testing, but I figured I'd clean it up.